lib/ab_test.rb

```diff
@@ -50,7 +64,16 @@ def initialize(
   # @param [Hash] session
   # @param [User] user
   # @param [Hash] user_session
-  def bucket(request:, service_provider:, session:, user:, user_session:)
+  # @param [Boolean] persisted_read_only Avoid new bucket assignment if test is configured to be
+  # persisted but there is no persisted value.
```

Comment:
this is that YARD thing we have checks for in the `analytics_events.rb` with line continuations
```suggestion
  # @param [Boolean] persisted_read_only Avoid new bucket assignment if test is configured to be
  #   persisted but there is no persisted value.
```

Reply:
> this is that YARD thing we have checks for in the `analytics_events.rb` with line continuations

Good catch! Updated in 54c4274.